docs/root/version_history/current.rst

```diff
@@ -1,5 +1,5 @@
-1.16.1 (Pending)
-================
+1.16.1 (November 20, 2020)
```

Comment:
Note all of these need to get merged to main at some point, so please make a note on that.